docs/ApiPrivateEndpointConnection.md

```diff
@@ -0,0 +1,108 @@
+# ApiPrivateEndpointConnection
```

Comment:
Do you know why it has these `Api*` prefixes on some of these types? Is the code generator trying to namespace the types or something?

Reply:
Yea, it's happening bc a `PrivateEndpointConnection` message also exists in `commonpb`, a dependency for `apipb` so the generator namespaces the type to disambiguate. It kinda sucks we have a dependency on commonpb, though from quick perusal, it seems to only be used for `KeysetPaginationRequest` and `SortOrder`, so we should be able to ditch the dependency relatively easily

Reply:
Ya that's pretty unfortunate. Since this is public-facing, I think we should probably try to fix it before cutting a release.

What's your timeline for this? Do you need this right away? Do you think you'd be able to move things around to avoid the collision?